Library/test-helpers/lib.sh

```diff
@@ -1253,6 +1253,78 @@ limeCreateTestDir() {
 }
 
 
+true <<'=cut'
+=pod
+
+=head2 limeCopySourceFile
+
+Copy file to actual dir from specified path or download file 
+from repository to actual dir.
+
+    limeCopySourceFile
+
+=over
+
+=back
+
+Returns 0 when the when copy of file to actual dir is succesfull.
+
+=cut
+
+limeCopySourceFile(){
+
+    local OPTION=$1
+    local FILEPATH=$2
+
+    if [ ${OPTION} == "--c" ]; then
+        cp ${FILEPATH} . && chmod a+x ${NAME}
+    elif [ ${OPTION} == "--i" ]; then
+        curl -O https://raw.githubusercontent.com/keylime/keylime/master/scripts/${NAME} && chmod a+x ${NAME}
+    fi
+
+}
+
+true <<'=cut'
+=pod
+
+=head2 limeGetSourceFile
```

Comment:
It should be the other way around. `limeGetFilePath [--source|-s|--installed|-i]` should only print the complete path of the found file. While the copy function should use that path to copy that file to a requested location.